### ISSUE TITLE
При импорте конфигурации общие модули оказываются пустыми

### ISSUE THREAD

**Issue by IojkinKot** [1C-Company/1c-edt-issues#1511]
### Описание ошибки

Импортирую конфигурацию, открывается окно сравнения/объединения текущего проекта с конфигурацией, где некоторые общие модули конфигурации оказываются пустыми.

### Как воспроизвести

Стабильного воспроизведения нет, но ошибка встречается довольно часто.

### Скриншоты

![image](https://github.com/user-attachments/assets/96e7d9dd-1e99-486c-91f3-d76072e33061)
![image](https://github.com/user-attachments/assets/0537c171-1d51-45e8-93ff-704432c3ab86)

### Ожидаемое поведение

Не показывать различия проекта и конфигурации, которых нет. Текст общих модулей должен быть.

### Лог рабочей области

[logs.zip](https://github.com/user-attachments/files/17024279/logs.zip)

### Версия 1С:EDT

1C:Enterprise Development Tools Ruby 2023.3.6

### Операционная система

Windows

### Версия платформы 1С:Предприятие 8

8.3.24.1667

### Установленные плагины

Нет плагинов

**Comment by IojkinKot** [comment]
Коллеги, поддержите issue, а то разработчики EDT почему-то думают, что это не критичный баг.

**Comment by LesN1ck** [comment]
Пока для себя заметил закономерность, что ошибка возникает чаще, если попеременно ходить между окнами сравнения модулей в окне сравнения/объединения и в панели "Индексирование Git". Например, если грузить конфигурацию и расширения. Или параллельно со сравнением/объединением выполнять еще какие-то сравнительные операции. Если есть возможность, теперь всегда сначала молча забираю всё (максимум, флажками в окне сравнения/объединения натыкать, что нужно), и уже потом по изменению индексируемых файлов смотрю, что поменялось.

**Comment by NikitaMikhaylovSB** [comment]
Проверьте на 2024.1.2 - там правили подобную ошибку.

**Comment by tretyakevich** [comment]
Да, пожалуйста обновите EDT и проверьте. В случае повторения - возьмем в работу.

**Comment by tormozit** [comment]
Проблема актуальна в EDT 2025.2. Но продолжаем в новой заявке https://github.com/1C-Company/1c-edt-issues/issues/2065
